spec/models/delete_service_event_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe DeleteServiceEvent, type: :model do
+  let(:service) { create(:service) }
+  let(:data) { { name: service.name, entity_id: service.entity_id } }
+  context 'on success' do
+    it 'event is persisted' do
+      delete_service_event = create(:delete_service_event, data: data)
+      expect(delete_service_event).to be_valid
+      expect(delete_service_event).to be_persisted
+    end
+  end
+  context 'on failure' do
+    it 'when name is not present' do
```

Comment:
we're not passing name or entity_id to the event, so they don't need to be validated/expected